docker-compose.yaml

```diff
@@ -43,7 +43,7 @@ services:
       - LOCAL_DOCKER_MOUNT=${LOCAL_DOCKER_MOUNT}
       - CONFIG_ROOT=${CONFIG_ROOT}
       - TRACKING_STRATEGY=${TRACKING_STRATEGY}
-      - AIRBYTE_VERSION=${VERSION}
+      - AIRBYTE_VERSION=${CODE_VERSION}${VERSION}
```

Comment:
per comment above is code_version necessary. i feel like this could get pretty dicey.

Reply:
Agreed, having two versions smells like a problem.

Reply:
After discussion, we found a way to avoid this use of two env variables!

This isn to the case anymore